docs/reference/commands/certutil.asciidoc

```diff
@@ -151,6 +166,9 @@ parameter cannot be used with the `ca` parameter.
 
 `-h, --help`:: Returns all of the command parameters.
 
+`--http`:: Generates a new certificate or certificate request for the {es} HTTP
+interface.
+
```

Comment:
This should be `http` (no `--`) because it's a command rather than an option.
I think that means we want to put it at the top of the parameters list (after ca, cert, csr) as well.

Reply:
Oops!  Thanks for catching this!